modules/command-engine/core/src/test/scala/surge/internal/domain/SurgeMessagePipelineSpec.scala

```diff
@@ -29,7 +29,6 @@ import surge.metrics.Metrics
 
 import scala.concurrent.duration._
 
-@Ignore
```

Comment:
Aah wait, actually just remembered another reason this was Ignored in an earlier PR. The logs end up flooded with this error message because something from this suite doesn't quite close cleanly:
```
2021-07-26 19:45:50,352 WARN  [kafka-producer-network-thread | producer-3] org.apache.kafka.clients.NetworkClient: [Producer clientId=producer-3] Connection to node 0 (localhost/127.0.0.1:6001) could not be established. Broker may not be available.
12315
```

Not sure if you have insights into what that might be?